go.mod

```diff
@@ -4,28 +4,58 @@ go 1.18
 
 require (
 	github.com/alecthomas/kong v0.5.1-0.20220516223738-0aaa4c11997b
+	github.com/billgraziano/dpapi v0.4.0
```

Comment:
>/billgraziano/dpapi v0.4.0 [](http://example.com/codeflow?start=11&length=26)

are we going to dump dpapi and just stored clear text?

Reply:
Going to encrypted bits to disk, then provide `sqlcmd config view --raw` to view the secret decrypted

Reply:
Made this an opt-in on windows, with the --encrypt-password switch